woocommerce/wc-admin
# Use wc filter to get status tabs for tools category

Fixes #6524 

Uses the status filter to get registered tabs for the tools menu.

Note that this requires copying over the default tabs, but I don't think this is a worse solution than what we had since we were manually adding all these items anyway.

### Screenshots
<img width="283" alt="Screen Shot 2021-03-05 at 9 34 42 AM" src="https://user-images.githubusercontent.com/10561050/110130034-95662d80-7d96-11eb-9757-725e454d2e4e.png">


### Detailed test instructions:

1. Register a new tab via the filter.
```
add_filter( 'woocommerce_admin_status_tabs', function ( array $tabs ) {
	$tabs['my-tools-page'] = __( 'My Tools Page', 'your-text-domain' );
	return $tabs;
} );
```

2. Enable the new navigation.
3. Make sure the menu item for the registered tab is shown under `Tools`.
